src/components/ScrollableContainer/ScrollableContainer.test.js

```diff
@@ -0,0 +1,112 @@
+              duis ea laborum ex. Reprehenderit consequat officia ea id ex
+              exercitation et sit et. Velit velit aliqua occaecat quis occaecat.
+              Enim incididunt est velit pariatur adipisicing labore dolore anim
+              cillum.
+            </p>
+            <p>
+              Ullamco reprehenderit in irure officia dolore anim eiusmod labore
+              duis ea laborum ex. Reprehenderit consequat officia ea id ex
+              exercitation et sit et. Velit velit aliqua occaecat quis occaecat.
+              Enim incididunt est velit pariatur adipisicing labore dolore anim
+              cillum.
+            </p>
+            <p>
+              Ullamco reprehenderit in irure officia dolore anim eiusmod labore
+              duis ea laborum ex. Reprehenderit consequat officia ea id ex
+              exercitation et sit et. Velit velit aliqua occaecat quis occaecat.
+              Enim incididunt est velit pariatur adipisicing labore dolore anim
+              cillum.
+            </p>
+            <p>
+              Ullamco reprehenderit in irure officia dolore anim eiusmod labore
+              duis ea laborum ex. Reprehenderit consequat officia ea id ex
+              exercitation et sit et. Velit velit aliqua occaecat quis occaecat.
+              Enim incididunt est velit pariatur adipisicing labore dolore anim
+              cillum.
+            </p>
+          </BodyUI>
+        }
+        footer={<FooterUI className="TESTING__FOOTER">Footer</FooterUI>}
+      />
+    )
+
+    const containerScroll = container.querySelector('.c-ScrollableContainer')
+    const header = container.querySelector('.ScrollableContainer__Header')
+    const body = container.querySelector('.ScrollableContainer__Body')
+    const footer = container.querySelector('.ScrollableContainer__Footer')
+
+    // Elements present
+    expect(containerScroll).toBeInTheDocument()
+    expect(header).toBeInTheDocument()
+    expect(body).toBeInTheDocument()
+    expect(footer).toBeInTheDocument()
+
+    // Passing custom classnames
+    expect(header.classList.contains('TESTING__HEADER')).toBeTruthy()
+    expect(body.classList.contains('TESTING__BODY')).toBeTruthy()
+    expect(footer.classList.contains('TESTING__FOOTER')).toBeTruthy()
+
+    // Correct layout styles
+    expect(window.getComputedStyle(containerScroll).display).toBe('flex')
+    expect(window.getComputedStyle(containerScroll).flexDirection).toBe(
+      'column'
+    )
+    expect(window.getComputedStyle(containerScroll).flexGrow).toBe('1')
+    expect(window.getComputedStyle(containerScroll).overflow).toBe('hidden')
+    expect(window.getComputedStyle(body).overflow).toBe('auto')
+    expect(window.getComputedStyle(body).flexGrow).toBe('1')
+    expect(window.getComputedStyle(header).zIndex).toBe('3')
+    expect(window.getComputedStyle(footer).zIndex).toBe('3')
+
+    // starter shadows
+    expect(header.getAttribute('style')).toBeFalsy()
```

Comment:
could be replaced with `expect(header).not.toHaveAttribute('style')` (and similar below)

Reply:
ditto